src/scene.c

```diff
@@ -3,6 +3,49 @@
 #include "custom/toonscape.h"
 #endif
 
+static void scene_prepare_texture(Scene *, int);
+static void scene_set_texture_pixels(Scene *, int);
+
+#ifdef RENDER_SW
+static void scene_texture128_scanline(int32_t *restrict,
+                                      int32_t *restrict,
+                                      int, int, int, int, int,
```

Comment:
would prefer to keep the prototype argument names in parity with implementation for consistency with the rest of the project and type hinting (even if it's static)